resources/js/views/settings/SettingsHome.vue

```diff
@@ -1,5 +1,75 @@
 <template>
-  <div v-html="$t('settings.description')">
+  <div>
+    <h3>{{ $t('settings.overview') }}</h3>
+    <p>{{ $t('settings.overviewDescription') }}</p>
+    <hr>
+    <b-row cols="1" cols-sm="2" cols-md="3" cols-lg="3" align-v="stretch">
+      <can method='viewAny' policy='SettingPolicy'>
```

Comment:
The Can-Component isn't imported so that the cards displayed even if the user hasn't the necessary permissions.